src/components/Slide.js

```diff
@@ -15,12 +15,18 @@ const Slide = props => {
   const { children, slideNum } = props;
 
   const numberOfSlideElements = Array.isArray(children)
-    ? children.filter(x => isComponentType(x, 'SlideElementWrapper')).length
+    ? children.filter(x => isComponentType(x, 'SlideElementWrapper')).length + 1
```

Comment:
What's the `+ 1` for?

Reply:
Ummmm I'm gonna remove it, thanks for picking it up!